packages/aws-lambda/layer/bin/publish-layer.sh

```diff
@@ -373,7 +375,13 @@ if [[ -z $SKIP_AWS_PUBLISH_LAYER ]]; then
           --output json \
           --compatible-runtimes nodejs18.x nodejs20.x \
           | jq '.Version' \
-      )
+      ) || true  # NOTE: If the upload fails, the bash script should not fail.
+
+      if [[ -z $lambda_layer_version ]] || [[ ! $lambda_layer_version =~ ^[0-9]+$ ]]; then
+        echo "   + ERROR: Layer upload failed for region $region. Skipping region."
+        BUILD_SHOULD_FAIL=1
```

Comment:
I'm sharing an idea that could be considered for a future PR or improvement regarding the China region handling. If a publish attempt fails, we could add the region to a `FAILED_REGIONS` list and establish a fresh connection to retry those failed regions at the end.

Reply:
👍 Yes. Feel free to raise a card <3

Reply:
Created, INSTA-19355